client/src/pages/Campaign.js

```diff
@@ -57,12 +86,44 @@ const Campaign = (props) => {
   const [templateId, setTemplateId] = useState(0);
   const [emailTemplates, setEmailTemplates] = useState([{}]);
   const [success, setSuccess] = useState(false);
+  const [currentView, setCurrentView] = useState('summary');
+  const [selectedProspects, handleSelectedProspects] = useState([]);
+  const [prospects, handleProspects] = useState([{}]);
 
   useEffect( () => {
     getCampaign();
     getEmailTemplates();
+    getAllProspects();
+  // eslint-disable-next-line react-hooks/exhaustive-deps

```

Comment:
Does this disable the react-hook warnings?

Reply:
yes, or else every time when I save, it auto-fill the empty array.